packages/paste-website/src/pages/components/breadcrumb/index.mdx

```diff
@@ -154,6 +154,20 @@ You may choose to use a page Heading as the current page in the breadcrumb.
 </Box>`}
 </LivePreview>
 
+### Breadcrumb with a router
+
+Using breadcrumb with a router is analogous to
+[using Anchor with a router](/components/anchor/#using-anchor-with-a-router), since
+our BreadcrumbItem component creates links using [Anchor](/components/anchor) under the hood.
+
+<iframe
+  src="https://codesandbox.io/embed/using-breadcrumb-with-react-router-t24en?fontsize=14&hidenavigation=1&theme=dark"
```

Comment:
In the deploy preview it looks like this is a duplicate sandbox of the anchor demo, except that the sandbox title is different. Is this intentional or should we show a demo/example of using `Breadcrumb`?

Reply:
They look nearly identical, but this is the one w/ breadcrumb (there just aren't any breadcrumbs on the Home page since it's a home page).

It would probably be more useful if what I linked was the specific Page 2 where there's the most breadcrumbs. Is there a way to do that in a Codesandbox embed?

Reply:
Figured out how to change the initial path, so upon opening this docs page it should show the breadcrumbs. Lemme know if this looks good to you @andipants12!